cypress/e2e/smokeTests/sharedTests/auth-tests.ts

```diff
@@ -40,6 +40,11 @@ function storeToken() {
   }
 }
 
+function hasSandboxZenodo() {
+  const baseUrl = Cypress.config('baseUrl');
+  return baseUrl === 'https://qa.dockstore.org' || baseUrl === 'https://staging.dockstore.org' || baseUrl === 'https://dev.dockstore.net';
```

Comment:
The develop branch runs of the smoke tests run against 1.15 prod, so this means we won't be restubbing on prod as soon as this is merged.

Is that the intent? It may be tricky to manage if we want to keep the tests running as is until 1.16 is deployed, so this could be OK, just making sure.

Reply:
> The develop branch runs of the smoke tests run against 1.15 prod, so this means we won't be restubbing on prod as soon as this is merged.
> 
> Is that the intent? It may be tricky to manage if we want to keep the tests running as is until 1.16 is deployed, so this could be OK, just making sure.

Ok with me, it's a side affect of our smoke testing scheme that it's probably not worth working around, in this case.